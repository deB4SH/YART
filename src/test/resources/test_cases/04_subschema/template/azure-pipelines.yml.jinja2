trigger:
  batch: true
  branches:
    include:
      - main
      - master

variables:
  - group: {{ global.devopsPipeline.variableGroupName }}

pool:
  name: {{ global.devopsPipeline.poolName }}

jobs:
  - job: deploy_kubernetes_cluster
    workspace:
      clean: all
    steps:
      - checkout: self
      - template: azure-pipelines/template-manifests.yaml
      - template: azure-pipelines/apply-cluster.yaml
        parameters:
          clusters:
          {%- for cluster in clusters %}
            - name: {{ cluster.name }}
              path: cluster/{{ cluster.name }}
              namespace: {{ cluster.supervisorNamespace }}
              tanzuSupervisorServer: $(tanzu-supervisor-server)
              tanzuAdUser: $({{ cluster.tanzuAdUserVar }})
              tanzuAdUserPw: $({{ cluster.tanzuAdUserPwVar }})
          {%- endfor %}