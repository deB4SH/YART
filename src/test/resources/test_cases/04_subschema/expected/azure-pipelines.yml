trigger:
  batch: true
  branches:
    include:
      - main
      - master

variables:
  - group: 

pool:
  name: 

jobs:
  - job: deploy_kubernetes_cluster
    workspace:
      clean: all
    steps:
      - checkout: self
      - template: azure-pipelines/template-manifests.yaml
      - template: azure-pipelines/apply-cluster.yaml
        parameters:
          clusters:
            - name: happy-path-cluster
              path: cluster/happy-path-cluster
              namespace: tanzu-svc
              tanzuSupervisorServer: $(tanzu-supervisor-server)
              tanzuAdUser: $()
              tanzuAdUserPw: $()
            - name: happier-path-cluster
              path: cluster/happier-path-cluster
              namespace: tanzu-svc
              tanzuSupervisorServer: $(tanzu-supervisor-server)
              tanzuAdUser: $()
              tanzuAdUserPw: $()